hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -338,32 +365,24 @@ public void testCreateDirectory() throws IOException {
     keyArgs = createBuilder()
         .setKeyName(keyName)
         .build();
-    OpenKeySession keySession = keyManager.openKey(keyArgs);
+    OpenKeySession keySession = writeClient.openKey(keyArgs);
     keyArgs.setLocationInfoList(
         keySession.getKeyInfo().getLatestVersionLocations().getLocationList());
-    keyManager.commitKey(keyArgs, keySession.getId());
+    writeClient.commitKey(keyArgs, keySession.getId());
     try {
-      keyManager.createDirectory(keyArgs);
+      writeClient.createDirectory(keyArgs);
       Assert.fail("Creation should fail for directory.");
     } catch (OMException e) {
       Assert.assertEquals(e.getResult(),
           OMException.ResultCodes.FILE_ALREADY_EXISTS);
     }
 
-    // create directory for root directory
-    keyName = "";
-    keyArgs = createBuilder()
-        .setKeyName(keyName)
-        .build();
-    keyManager.createDirectory(keyArgs);
-    Assert.assertTrue(keyManager.getFileStatus(keyArgs).isDirectory());
-
```

Comment:
I removed this test because I think it is invalid.  See [OMDirectoryCreateRequest](https://github.com/apache/ozone/blob/c4579ec5427758f6fa53b555b8de05a9d3f4d222/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMDirectoryCreateRequest.java#L175)